Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -70,13 +70,21 @@
             "name": "WebAppKind",
             "type": "pickList",
             "label": "Web app kind",
-            "defaultValue": "app",
+            "defaultValue": "webApp",
             "required": false,
-            "visibleRule": "WebAppName = invalidwebappname",
+            "options": {
+                "webApp": "Web App",
+                "functionApp": "Function App",
+                "logicApp": "Logic App",
+                "apiApp": "API App",
```

Comment:
Doesn't the logic require any change? I don't see any changes in any js file - is that expected?